lib/GameObjects/Unit.js

```diff
@@ -31,6 +31,39 @@ function Unit(stats) {
     this.action = null;
     this.source = null;
     this.target = null;
+
+    //let's calculate some defeat points!
+    var dp = this.maxHp / this.movementDelay;
```

Comment:
Need to make sure not to put movementDelay of 1000 or dp will be almost nothing.

Might want to repalce it with Math.min(100, this.movementDelay)